applications/nrf5340_audio/src/bluetooth/bt_management/dfu/bt_mgmt_dfu.c

```diff
@@ -73,6 +73,10 @@ static struct bt_conn_cb dfu_conn_callbacks = {
 	.recycled = dfu_recycled_cb,
 };
 
+/*
+ * If desired, the different names can be set in DFU mode.
+ * E.g. based on the device location.
+ */
```

Comment:
Revise this section. Unclear to me what is meant here. As far as I can see the device location is no longer part of the DFU name

Reply:
Agreed. Have removed the comment.